### PR TITLE
Add annotation + viola + join checks + cleanup

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-structural/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: bin/simple-event-annotation.R
Comment (mvheetve) [inline bin/simple-event-annotation.R:1]: this is the R script for gridss compatibility issues?

File: assets/samplesheet.csv [hunk 1]
```diff
@@ -1,6 +1,5 @@
 sample,family,cram,crai,bed
 PosCon1,family1,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon1.cram,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon1.cram.crai,s3://test-data/genomics/homo_sapiens/illumina/regions/SVcontrol/PosCon1and2.roi.bed
 PosCon2,family1,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon2.cram,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon2.cram.crai,s3://test-data/genomics/homo_sapiens/illumina/regions/SVcontrol/PosCon1and2.roi.bed
-PosCon3,,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon3.cram,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon3.cram.crai,s3://test-data/genomics/homo_sapiens/illumina/regions/SVcontrol/PosCon3.roi.bed
+PosCon3,,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon3.cram,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon3.cram.crai,
```
[the remark this answers is not in the document]
Comment (nvnieuwk) [reply assets/samplesheet.csv:4]: For testing purposes :) mainly to test if the pipeline can also run without an ROI file

File: bin/simple-event-annotation.R
Comment (nvnieuwk) [reply bin/simple-event-annotation.R:1]: Yes but it doesn't work ATM

File: subworkflows/local/bam_variant_calling_gridss/main.nf [hunk 2]
```diff
@@ -2,15 +2,16 @@
...
-include { GRIDSS_GRIDSS      } from '../../../modules/nf-core/gridss/gridss/main'
+include { SIMPLE_EVENT_ANNOTATION   } from '../../../modules/local/gridss/simple_event_annotation/main'
 
-include { TABIX_TABIX        } from '../../../modules/nf-core/tabix/tabix/main'
+include { GRIDSS_GRIDSS             } from '../../../modules/nf-core/gridss/gridss/main'
+include { TABIX_TABIX               } from '../../../modules/nf-core/tabix/tabix/main'
 
 workflow BAM_VARIANT_CALLING_GRIDSS {
     take:
         crams                   // channel: [mandatory] [ meta, cram, crai ] => The aligned CRAMs per sample with the regions they should be called on
```
Comment (mvheetve) [inline subworkflows/local/bam_variant_calling_gridss/main.nf:12]: why cram and not ch_cram like for manta and delly?

File: subworkflows/local/bam_variant_calling_whamg/main.nf [hunk 3]
```diff
@@ -13,7 +13,7 @@ workflow BAM_VARIANT_CALLING_WHAMG {
         crams                   // channel: [mandatory] [ meta, cram, crai ] => The aligned CRAMs per sample with the regions they should be called on
         beds                    // channel: [optional]  [ meta, bed, bed_gz, bed_gz_tbi ] => A channel containing the normal BED, the bgzipped BED and its index file
         fasta                   // channel: [mandatory] [ fasta ] => The fasta reference file
-        fasta_fai               // channel: [mandatory] [ fasta_fai ] => The index of the fasta reference file
+        fai               // channel: [mandatory] [ fai ] => The index of the fasta reference file
```
Comment (mvheetve) [inline subworkflows/local/bam_variant_calling_whamg/main.nf:16]: also here, why not ch_xxx?

**Review by mvheetve — APPROVED** [review]
I have a feeling some of the main.nf files are not in the uniform format yet (ch_XXXX, val_XXXX, etc), but this can be added later. I approve for now.

File: subworkflows/local/bam_variant_calling_gridss/main.nf (hunk 2 above)
Comment (nvnieuwk) [reply subworkflows/local/bam_variant_calling_gridss/main.nf:12]: The gridss and whamg subworkflows are a WIP and aren't used currently. (Basically me being a bit lazy and only updating the code that's actually being used at the moment 😁)

**Pull request merged**